fs/ensure_symlink.ts

```diff
@@ -4,15 +4,28 @@ import { ensureDir, ensureDirSync } from "./ensure_dir.ts";
 import { getFileInfoType, toPathString } from "./_util.ts";
 import { isWindows } from "../_util/os.ts";
 
+export function resolveSymlinkSrc(src: string, dest: string): string;
+export function resolveSymlinkSrc(src: URL, dest: URL): URL;
+export function resolveSymlinkSrc(src: string | URL, dest: string | URL) {
```

Comment:
This looks like an internal utility. So Let's not export this?

Reply:
ok